test/integration/preflight/check-yum-update/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:7
+RUN yum install -y rpm-build rpmdevtools createrepo && \
+    rpmdev-setuptree && \
+    mkdir -p /mnt/localrepo
```

Comment:
Assuming we're building this image on the fly and never shipping it, size doesn't much matter, and I think I'd like to have the metadata cached so each container runs the test faster. It probably doesn't matter a whole lot in the end though.